tests/lib/rules/jsx-prefer-fragment-wrappers.js

```diff
@@ -0,0 +1,41 @@
+const {RuleTester} = require('eslint');
+const rule = require('../../../lib/rules/jsx-prefer-fragment-wrappers');
+
+const ruleTester = new RuleTester();
+const parserOptions = {ecmaVersion: 6, ecmaFeatures: {jsx: true}};
+
+function errorWithTagName(tagName) {
+  return [
+    {
+      type: 'JSXElement',
+      message: `replace wrapping ${tagName} with fragment shorthand`,
+    },
+  ];
+}
+
+ruleTester.run('jsx-prefer-fragment-wrappers', rule, {
+  valid: [
+    {code: '<div className={className}>{foo}{bar}<Baz /></div>', parserOptions},
+    {code: '<div><Bar /></div>', parserOptions},
```

Comment:
This also seems useless?

Reply:
@lemonmade Yeah I suppose it is, but I would argue that this is still valid for this rule. Right now, I'm checking for multiple children because we are trying to prevent the case where a developer will add a wrapping `<div>` instead of a Fragment. If we are just checking for useless markup, that might be a different rule or at least a different error message.